docs/design-system/README.md

```
@@ -0,0 +1,526 @@
+  1.2s shimmer keyframes.
+- **Error**: `.error-text` paragraph in `--error`; inline report errors
+  show inside the report panel, not as a toast.
+- **Last-updated chip**: `.at-muted` text right-aligned in panel header.
+
+---
+
+## Page patterns
+
+### Screener (`screener_dashboard.html`)
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)